src/custom/hooks/usePriceImpact/index.ts

```diff
@@ -1,6 +1,6 @@
-import { useEffect, useRef, useState } from 'react'
+import { useEffect, useState } from 'react'
```

Comment:
u created the index for adding the tests next to the hook, but I don't fully understand, u don't plan to test the hook but just the function itself, right? 

If that's the case, the function should be a pure function in a util and the test should be for that util not the hook

Reply:
just function, i can move it

Reply:
done